Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayAuthorizationDelegate.m

```diff
@@ -177,9 +174,39 @@ - (void)updateCheckoutWithAddressCompletion:(void (^)(PKPaymentAuthorizationStat
 		[self.client updateCheckout:self.checkout completion:^(BUYCheckout *checkout, NSError *error) {
 			if (checkout && error == nil) {
 				self.checkout = checkout;
-				[self getShippingRates:self.checkout completion:completion];
-			}
-			else {
+				
+				if ([self.checkout requiresShipping] == NO) {
+					completion(PKPaymentAuthorizationStatusSuccess, nil, [self.checkout buy_summaryItemsWithShopName:self.shopName]);
+				} else {
+					
+					self.shippingRates = @[];
+					[self.client getShippingRatesForCheckoutWithToken:self.checkout.token completion:^(NSArray *shippingRates, BUYStatus status, NSError *error) {
+						
+						self.shippingRates = shippingRates;
+						if (shippingRates) {
```

Comment:
We don't need this conditional check—the `-buy_convertShippingRatesToShippingMethods:` will return an empty array if we pass in nil.